osimpipeline/task.py

```diff
@@ -96,29 +105,183 @@ def __init__(self, trial):
         super(TrialTask, self).__init__(trial.subject)
         self.trial = trial
 
+class SetupTask(TrialTask):
+    def __init__(self, tool, trial, 
+                 model_to_adjust=None, 
+                 adjusted_model=None,
+                 gait_cycles='concatenated'):
+        super(SetupTask, self).__init__(trial)
+        self.tool = tool
+        self.trial = trial
+        self.name = '%s_%s_setup' % (trial.id, self.tool)
+        self.tool_path = os.path.join(trial.results_exp_path, self.tool)
+        self.doit_path = self.study.config['doit_path']
+        self.gait_cycles=gait_cycles
+        self.source_extloads_fpath = os.path.join(self.trial.rel_path, 
+                self.tool, 'external_loads.xml')
```

Comment:
> I cannot easily run Sam Hamner's and Edith Arnold's setup files on my computer because they used relative paths

Did you mean Sam and Edith didn't use relative paths? Either way, I'm addressing the relative paths issue now and I am running into the issue with the external loads.